crates/sui-rpc-api/proto/sui.types.proto

```diff
@@ -0,0 +1,1040 @@
+syntax = "proto3";
+
+package sui.types;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+
+message Address {
+  bytes address = 1;
+}
+
+message ObjectId {
+  bytes object_id = 1;
+}
+
+message Digest {
+  bytes digest = 1;
+}
+
```

Comment:
Probably should add a comment at the top of this file. This is a 1:1 mapping of Sui public data types to protobuf definitions.